docs/static-s3-website.md

```diff
@@ -0,0 +1,78 @@
+## Construct StaticS3Website
+
+This construct creates an S3 bucket and upload any content into it.
+It is a composition of the following CDK constructs:
+    - [AWS S3 Bucket lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3)
+    - [AWS S3 Bucket deployment lib](https://github.com/aws/aws-cdk/tree/main/packages/aws-cdk-lib/aws-s3-deployment)
+
+Why to use it over the CDK constructs?
```

Comment:
discussion: I would remove this paragraph as there isn't another construct with a similar behavior, so this is not "better", it's simply a new composition.